apple/RNSVGRenderable.mm

```diff
@@ -257,42 +257,38 @@ - (void)renderTo:(CGContextRef)context rect:(CGRect)rect
 #endif
     }
 #endif // TARGET_OS_OSX
-    NSUInteger iheight = (NSUInteger)height;
-    NSUInteger iwidth = (NSUInteger)width;
-    NSUInteger iscale = (NSUInteger)scale;
-    NSUInteger scaledHeight = iheight * iscale;
-    NSUInteger scaledWidth = iwidth * iscale;
-    NSUInteger npixels = scaledHeight * scaledWidth;
-    CGRect drawBounds = CGRectMake(0, 0, width, height);
-
-    // Allocate pixel buffer and bitmap context for mask
-    NSUInteger bytesPerPixel = 4;
-    NSUInteger bitsPerComponent = 8;
-    NSUInteger bytesPerRow = bytesPerPixel * scaledWidth;
-    CGColorSpaceRef colorSpace = CGColorSpaceCreateDeviceRGB();
+    // TODO: make it the right way, currently this is a hack to get the correct scale for the mask
+    scale = scale * self.matrix.a * self.svgView.viewBoxTransform.a;
```

Comment:
Can we get more explanation why we do it this way?

Reply:
I believe there is currently no way to directly obtain the end scale of the current element, so we calculate it a bit silly way as: `screen scale * current element scale * viewBox scale`